prow/istio-pilot-e2e.sh

```diff
@@ -56,4 +56,4 @@ HUB="gcr.io/istio-testing"
 
 cd ${GOPATH}/src/istio.io/istio
 ./bin/init.sh
-make -C "${ROOT}/pilot" e2etest HUB="${HUB}" TAG="${GIT_SHA}" TESTOPTS="-mixer=false -use-sidecar-injector=false -use-admission-webhook=false"
+make e2e_pilot HUB="${HUB}" TAG="${GIT_SHA}" TESTOPTS="-mixer=false -use-sidecar-injector=false -use-admission-webhook=false"
```

Comment:
shouldn't this change be reflected in circleci as well?

Reply:
I'm not sure I understand the question, but I read it as suggesting that circleCI can be changed to use "make e2e_pilot" rather than launch bin/pilot-integration-test directly.  I can give that a try.  It looks like prow and circleCI use a different set of parameters, though I don't know if that's intentional or accidental (circleCI uses true for -mixer while prow uses -false.  CircleCI also has different sections for -auth with enable vs disable and prow doesn't specify anything).

Reply:
Prow should use mixer true. And yes I meant we should sync them up.